packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.styles.ts

```diff
@@ -107,7 +107,12 @@ export const getStyles = (props: IProgressIndicatorStyleProps): IProgressIndicat
             position: 'absolute',
             minWidth: '33%',
             background: `linear-gradient(to right, ${progressTrackColor} 0%, ${palette.themePrimary} 50%, ${progressTrackColor} 100%)`,
-            animation: `${isRTL ? IndeterminateProgressRTL : IndeterminateProgress} 3s infinite`
+            animation: `${isRTL ? IndeterminateProgressRTL : IndeterminateProgress} 3s infinite`,
+            selectors: {
+              [HighContrastSelector]: {
```

Comment:
I dont think we need this here? having background color set above should cover this case.

Reply:
the existing gradient looks pretty bad in high contrast. this feels much more intentional than an oddly rendered gradient. It doesn't use a system color, but the actual primary blue. And the gradient chops off quite quickly
 
![progress2](https://user-images.githubusercontent.com/1434956/70817102-b1db0c00-1d85-11ea-8d72-c3ac7263d8fc.gif)